src/screens/Media/Media.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { Suspense, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router';
+
+import useMedia from '#src/hooks/useMedia';
+import ErrorPage from '#src/components/ErrorPage/ErrorPage';
+import { getCustomScreen } from '#src/utils/common';
+import { mediaScreens } from '#src/customScreens';
```

Comment:
I always find it hard to read / debug when a lookup dictionary is created and stored apart from where it's primarily used. It would be nice to have this more accessible and coupled to this component.

Reply:
The reason this is dictionary lives outside the component is to prevent publishers/developers from needing to search for the screenMapping. It now lives at a central location. Perhaps when combining media and playlist, it doesn't matter that much...

Reply:
> Perhaps when combining media and playlist, it doesn't matter that much...
Agreed